tests/utils/test_dates.py

```diff
@@ -29,8 +29,10 @@
 
 class TestDates(unittest.TestCase):
     def test_days_ago(self):
+        from airflow.settings import TIMEZONE
+
         today = pendulum.today()
-        today_midnight = pendulum.instance(datetime.fromordinal(today.date().toordinal()))
+        today_midnight = pendulum.instance(datetime.fromordinal(today.date().toordinal()), tz=TIMEZONE)
```

Comment:
I think this has different meanings in some edge case because `pendulum.today()` has some weird semantics if the local time is at a different date from UTC (e.g. at 2am in UTC+3, UTC is still on the previous day).